src/functions.c

```diff
@@ -396,6 +424,12 @@ void functionsHelpCommand(client *c) {
 "    In addition, returns a list of available engines.",
 "KILL",
 "    Kill the current running function.",
+"FLUSH [ASYNC|SYNC]",
+"    Flush all the functions.",
```

Comment:
```suggestion
"    Delete all the functions.",
```

In the docs for FLUSHDB and FLUSHALL, we say "remove" or "delete".

Flushing is not always the same as deleting, e.g. flushing an output buffer is not the same as deleting it. Here, it actually means delete, but we use the command name "flush" by tradition (analogy with other commands).